examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -47,11 +47,14 @@
     PHYSICAL_CHANNEL = sys.argv[3]
 
 
-async def _main():
+def _main():
     client = None
     task = None
 
-    async with grpc.aio.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}") as channel:
+    with (
+        grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}") as channel,
+        concurrent.futures.ThreadPoolExecutor() as executor,
+    ):
```

Comment:
Hmm, ok
```
Run ni-python-styleguide lint source/codegen/*.py examples
examples/nidaqmx/analog-input-every-n-samples.py:55:6[7](https://github.com/ni/grpc-device/actions/runs/5008134058/jobs/8975642212?pr=923#step:5:8): E999 SyntaxError: invalid syntax
examples/nidaqmx/analog-input-every-n-samples-aio.py:25:101: W505 doc line too long (101 > 100 characters)
Error: Process completed with exit code 1.
```

Reply:
https://docs.python.org/3/reference/compound_stmts.html#the-with-statement

> Changed in version 3.10: Support for using grouping parentheses to break the statement in multiple lines.

Reply:
That pipeline is still using `3.7`. Should be `3.9` right?

I guess the new syntax would break `mypy` (run by `validate_examples`, for which we exclude daq) but not `styleguide`? Or maybe not even `mypy`.

I think having this example require `3.10` is fine, even if `3.9` is the de facto version. Anyone that's interested in this is likely on newer python or can find their way around. And this syntax is useful, especially for this kind of code.

Reply:
Python 3.7 reaches end-of-life next month, but 3.8 and 3.9 will be around for a while: https://devguide.python.org/versions/#versions

The strange thing is I've been testing with 3.9, so I'm not sure if the version listed in the docs is correct.

Anyway, I don't think this is worth breaking compatibility with 3.8 and 3.9, so I'll update the example.